src/web/app/components/loading-spinner/loading-spinner.component.scss

```diff
@@ -0,0 +1,13 @@
+.loading-container {
+  margin: 0;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  -ms-transform: translate(-50%, -50%);
```

Comment:
I think `-ms-` is applied to internet explorer right? Not necessary?

Reply:
> What about when it hasn't been determined yet, we just leave the top right-hand corner empty?

That works, too.

Reply:
Actually, we're not likely to need any auto-prefixed version. Angular CLI comes with [Autoprefixer built-in](https://angular.io/guide/build#configuring-browser-compatibility).

Reply:
Got it!